src/Previews.tsx

```diff
@@ -97,7 +97,7 @@ export class Previews extends React.Component<PreviewsProps, PreviewsState> {
           {this.state.previews.map(
                 (preview: string, index: number) => (
                   <TabPanel value={this.state.value} index={index} key={index}>
-                    <img src={`${process.env.REACT_APP_PREVIEW_SERVER}/${this.props.runUid}/${preview}`} alt={"preview"} width="100%" />
+                    <img src={`${process.env.REACT_APP_PREVIEW_SERVER}/${this.props.runUid}/${preview}?${Math.random()}`} alt={"preview"} width="100%" />
```

Comment:
Why is this needed?

Reply:
I think this forces it to not use the cached image,